ts/components/ContextualHelpComponent.tsx

```diff
@@ -33,50 +33,85 @@ const ContextualHelpComponent: React.FunctionComponent<Props> = ({
-        avoidNavigationEventsUsage: true
-      }}
-      headerTitle={I18n.t("contextualHelp.title")}
-      customRightIcon={{
-        iconName: "io-close",
-        onPress: onClose,
-        accessibilityLabel: I18n.t("global.accessibility.contextualHelp.close")
-      }}
-    />
+}) => {
+  // Being content a ReactNode we can rely on `null` to represent no-content
+  const showLoader = contextualHelpData.content === undefined;
 
-    {!contextualHelpData.content && (
-      <View centerJustified={true}>
-        <ActivityIndicator color={themeVariables.brandPrimaryLight} />
-      </View>
-    )}
-    {contextualHelpData.content && (
-      <Content contentContainerStyle={styles.contentContainer} noPadded={true}>
-        <H3 accessible={true}>{contextualHelpData.title}</H3>
-        <View spacer={true} />
-        {contextualHelpData.content}
-        <View spacer={true} />
-        {contextualHelpData.faqs && isContentLoaded && (
-          <FAQComponent
-            onLinkClicked={onLinkClicked}
-            faqs={contextualHelpData.faqs}
-          />
-        )}
-        {isContentLoaded && (
-          <>
-            <View spacer={true} extralarge={true} />
-            <InstabugAssistanceComponent
-              requestAssistance={onRequestAssistance}
+  return (
+    <>
+      <BaseHeader
+        accessibilityEvents={{
+          avoidNavigationEventsUsage: true
+        }}
+        headerTitle={I18n.t("contextualHelp.title")}
+        customRightIcon={{
+          iconName: "io-close",
+          onPress: onClose,
+          accessibilityLabel: I18n.t(
+            "global.accessibility.contextualHelp.close"
+          )
+        }}
+      />
+      {showLoader && (
+        <View centerJustified={true}>
+          <ActivityIndicator color={themeVariables.brandPrimaryLight} />
+        </View>
+      )}
+      {!showLoader && (
+        <Content
+          contentContainerStyle={styles.contentContainer}
+          noPadded={true}
+        >
+          {renderTitleIfPresent(contextualHelpData.title)}
+          {renderContentIfPresent(contextualHelpData.content)}
+
+          {contextualHelpData.faqs && isContentLoaded && (
+            <FAQComponent
+              onLinkClicked={onLinkClicked}
+              faqs={contextualHelpData.faqs}
             />
-          </>
-        )}
-        {isContentLoaded && <EdgeBorderComponent />}
-      </Content>
-    )}
-    <BetaBannerComponent />
-  </>
-);
+          )}
+          {isContentLoaded && (
+            <>
+              {contextualHelpData.content !== null && (
+                <View spacer={true} extralarge={true} />
+              )}
+              <InstabugAssistanceComponent
+                requestAssistance={onRequestAssistance}
+              />
+            </>
+          )}
+          {isContentLoaded && <EdgeBorderComponent />}
+        </Content>
+      )}
+      <BetaBannerComponent />
+    </>
+  );
+};
+
+function renderTitleIfPresent(title: string) {
```

Comment:
This suggestion is only about code style. The code, from my POV, is correct.
Take it if it makes sense for you

```typescript
const renderTitleIfPresent = (title: string): React.ReactNode | null =>
  !isStringNullyOrEmpty(title) && (
    <>
      <H3 accessible={true}>{title}</H3>
      <View spacer={true} />
    </>
  );
```

- we often use `const` function with arrow syntax
- if / else is omitted (for my eye is more readable)
- add type also on function return (increase readability)

Reply:
Agree with (1) and (3) though I have an opposite view on (2) 😆 

That said, I'm going to follow your [suggestion below](https://github.com/pagopa/io-app/pull/3213#discussion_r666247914) and bring them back to the main body 🧱 

See https://github.com/pagopa/io-app/pull/3213/commits/0a11bdb086f68234faa4c4fcf585280488c764aa
I've kept the React wrapper `<></>` to ensure that the spacers stick to their content.